bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1331,11 +1352,18 @@
 <dust_flux_input%file_varname>DSTSF</dust_flux_input%file_varname>
 <dust_flux_input%scale_factor>1.0e-1</dust_flux_input%scale_factor>
 
-<iron_flux_source ocn_coupling="partial">monthly-calendar</iron_flux_source>
-<iron_flux_source                       >driver-derived  </iron_flux_source>
+<iron_flux_source>driver-derived</iron_flux_source>
+<iron_flux_source ocn_coupling="partial" ocn_bgc_config="cesm2.0">monthly-calendar</iron_flux_source>
 
 <dust_ratio_thres>55.0</dust_ratio_thres>
 <dust_ratio_thres ocn_bgc_config="cesm2.0">60.0</dust_ratio_thres>
+<dust_ratio_thres ocn_coupling="partial">66.2818</dust_ratio_thres>
```

Comment:
`get_value` in CIME's [utils/perl5lib/Build/NamelistDefaults.pm](https://github.com/ESMCI/cime/blob/8bffca6d3ccf9f2d47cf130e852d095ecff8378a/utils/perl5lib/Build/NamelistDefaults.pm#L232-L233) returns the matching entry with the most attributes. If there is a tie, it returns the first. So the `ocn_bgc_config="cesm2.0
` entry has precedence over the `ocn_coupling="partial"` entry, since it is first. We have other blocks with orderings like this (e.g., https://github.com/ESCOMP/POP2-CESM/blob/3d0472a59315ba54a17162404179d412a46aac73/bld/namelist_files/namelist_defaults_pop.xml#L203-L211), so I feel comfortable leaving this as is.